src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+import java.util.Random;
+
+public class ColorSupplier extends Ball {
+
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(ColorList.values().length);
+        ColorList randomColor = ColorList.values()[index];
```

Comment:
Otherwise in place of usage it looks strange when you have single random color but type is whole ColorList
```suggestion
        Color randomColor = Color.values()[index];
```